src/app/containers/Figure/fixtureData.jsx

```diff
@@ -69,35 +70,63 @@ const serviceContextStubNews = {
   imageCaptionOffscreenText: 'Image caption, ',
 };
 
-const generateFixtureData = (caption, copyright) => (
+const generateFixtureData = (caption, copyright, platform = 'canonical') => (
   <ServiceContext.Provider value={serviceContextStubNews}>
-    <FigureContainer
-      alt={imageAlt}
-      captionBlock={caption}
-      copyright={copyright}
-      height={imageHeight}
-      ratio={imageRatio}
-      src={imageSrc}
-      width={imageWidth}
-    />
+    <PlatformContext.Provider value={platform}>
+      <FigureContainer
+        alt={imageAlt}
+        captionBlock={caption ? captionBlock : null}
+        copyright={copyright ? copyrightText : null}
+        height={imageHeight}
+        ratio={imageRatio}
+        src={imageSrc}
+        width={imageWidth}
+      />
+    </PlatformContext.Provider>
   </ServiceContext.Provider>
 );
 
 export const FigureImage = generateFixtureData();
 
+export const FigureAmpImage = generateFixtureData(null, null, 'amp');
```

Comment:
This feels a bit wrong - a better approach IMO would be to pass an object:

```js
export const FigureAmpImage = generateFixtureData({ platform:  'amp' });
```

The function definition wouldn't have to change much:

```js
const generateFixtureData = ({ caption, copyright, platform = 'canonical'} ) => (
```